voltixApp/VoltixApp/Views/Vault/VaultAssetsView.swift

```diff
@@ -115,7 +115,7 @@ public struct VaultAssetsView: View {
     }
 	
     private func fetchCryptoPrices() async {
-        await CryptoPriceService.shared.fetchCryptoPrices(appState.currentVault)
+        await CryptoPriceService.shared.fetchCryptoPrices()
```

Comment:
We better pass in `vault` , because appState.currentVault might not be used in the future

Reply:
I am using the ApplicationState.shared singleton